Documentation/docs-site/content/usage/executor.en.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Controlling Function Execution"
+draft: false
+weight: 43
+---
+
+### Pool executor, low latency
```

Comment:
In this usage guide we shouldn't talk about the executor in the subheadings, only the use case. E.g.  

"Functions with no cold start overhead"
"Functions with no idle cost"
"Autoscaling"